src/utils/source-editor.js

```diff
@@ -40,8 +53,10 @@ class SourceEditor {
     return this.editor.getValue();
   }
 
-  setMode(value) {
-    this.editor.setOption("mode", value);
+  setMode(contentType) {
```

Comment:
hmm - this sadly won't work due to the fact that there's a different panel source editor.

We'll need to use https://github.com/devtools-html/debugger.html/blob/master/src/utils/source.js